Library/Homebrew/os/mac.rb

```diff
@@ -41,6 +41,10 @@ def cat
       version.to_sym
     end
 
+    def language
+      @language ||= Utils.popen_read('defaults', 'read', '.GlobalPreferences', 'AppleLanguages').delete(" \n\"()").sub(/,.*/, '')
```

Comment:
We mass-fixed core code pre-separation, so a while ago. There is stuff Rubocop skips over automatically fixing  though, as well as us having changed various Rubocop rules between then & now.